docker/kratos/config/kratos.yml

```diff
@@ -64,7 +64,7 @@ secrets:
 
 session:
   cookie:
-    domain: siasky.xyz
+    domain: secure.siasky.xyz
```

Comment:
We could use a comment here explaining why it should be `secure.siasky.xyz` and not `siasky.xyz`.

Hmmm... Does this mean that everyone running `accounts` will need to setup a subdomain? I guess so...